csharp/rtl/sdkrtl.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
should that be Looker...?

Reply:
Good question! I'm not sure. Something to check on. Thanks.